pkg/operator/ceph/cluster/cluster.go

```diff
@@ -136,8 +136,9 @@ func (c *cluster) detectCephVersion(image string, timeout time.Duration) (*cephv
 	return version, nil
 }
 
-func (c *cluster) createInstance(rookImage string, cephVersion cephver.CephVersion) error {
+func (c *cluster) createInstance(rookImage string, cephVersion cephver.CephVersion, state cephv1.ClusterState, name string) (bool, error) {
```

Comment:
- Can we always set this to the "Creating" state when the orchestration is starting? Then we don't need to pass this new `state` arg
- Since the `name` of the CR never changes, how about if we add it to the `cluster` struct so it doesn't need to be passed around